kratos/utilities/cpp_tests_utilities.h

```diff
@@ -67,6 +66,12 @@ namespace CppTestsUtilities
         const bool Elements = true
         );
 
+    /**
+     * @brief This method creates a pure (Element) simple geometry in 2D (triangles)
+     * @param rModelPart Reference to the ModelPart containing the problem
+     */
+    void KRATOS_API(KRATOS_CORE) CreatePure2DGeometry(ModelPart& rModelPart);
```

Comment:
```suggestion
    void CreatePure2DGeometry(ModelPart& rModelPart);
```
I think you don't need this as we're not exporting this to Python.

Reply:
Okay